manifests/canary-checker.yaml

```diff
@@ -152,8 +156,8 @@ spec:
         com.flanksource.infra.logs/processors.1.drop_event.when.contains.message: '[pod/canary] request completed with 503, expected [200 201 202], retrying'
         com.flanksource.infra.logs/processors.2.drop_event.when.contains.message: Requeue reconciliation
         com.flanksource.infra.logs/processors.3.drop_event.when.contains.message: Successfully Reconciled
-        cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
```

Comment:
Don't we need to keep this ?

Reply:
Yes, fixed

Reply:
it moved to the top of the annotation list.  I does need to be templated to be false if local-path storage is specified for the DB though, else we will lose the data...

Reply:
Added the condition